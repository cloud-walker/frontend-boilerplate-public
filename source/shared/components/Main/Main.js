import React from 'react'
import {Match, Miss} from 'react-router'
import Home from 'shared/pages/Home'
import Styleguide from 'shared/pages/Styleguide'
import NotFound from 'shared/pages/NotFound'

const Component = () =>
  <div>
    <Match exactly pattern="/" component={Home}/>
    <Match pattern="/styleguide" component={Styleguide}/>
    <Miss component={NotFound}/>
  </div>

Component.displayName = 'Main'

export default Component
