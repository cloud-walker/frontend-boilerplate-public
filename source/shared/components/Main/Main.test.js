import React from 'react'
import {shallow} from 'enzyme'
import Main from './index'

test('<Main/>', () => {
  expect(shallow(<Main/>)).toMatchSnapshot()
})
