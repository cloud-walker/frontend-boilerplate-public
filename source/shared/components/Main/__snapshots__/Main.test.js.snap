exports[`test <Main/> 1`] = `
<div>
  <Match
    component={[Function]}
    exactly={true}
    pattern="/" />
  <Match
    component={[Function]}
    exactly={false}
    pattern="/styleguide" />
  <Miss
    component={[Function]} />
</div>
`;
